src/components/Announcement.tsx

```diff
@@ -53,6 +56,17 @@ export default function () {
       <div className={classnames(flex('flex-1'))} />
       <AccentText small bold color="text-formal-accent">
         {announcementText}
+        <a
+          href="/app"
+          className={announcementLink}
+          onClick={() => {
+            setTimeout(() => {
+              announcementStore.announcementClosed = true
+            }, 75)
```

Comment:
Is it absolutely necessary to use the timeout here?

Reply:
No timeout needed, good catch